src/java/org/apache/cassandra/serializers/TypeSerializer.java

```diff
@@ -55,11 +60,45 @@ public final void validate(ByteBuffer bytes) throws MarshalException
 
     public abstract Class<T> getType();
 
-    public String toCQLLiteral(ByteBuffer buffer)
+    protected String toCQLLiteralNonNull(ByteBuffer buffer)
```

Comment:
nit: Maybe add `@Nonnull` to parameter?

Reply:
made this to param and return for each new type